packages/loopring_v3/contracts/lib/BytesUtil.sol

```diff
@@ -65,6 +65,20 @@ library BytesUtil
         return uint16(bytesToUintX(b, offset, 2) & 0xFFFF);
     }
 
+    function bytesToBytes4(
```

Comment:
I think we should make this function more gereric, like sliceBytes(offset, size)...

Reply:
The reason for the specific `bytesToBytes4` is that otherwise we would always have to cast to `bytes4` after calling a more general function. I modified it to make use of a more general function internally (similar to the uint functions).